pages/common/bats.md

```diff
@@ -0,0 +1,20 @@
+# bats
+
+> Bash Automated Testing System is a [TAP](https://testanything.org/)-compliant testing framework for Bash.
```

Comment:
```suggestion
> Bash Automated Testing System is a TAP-compliant testing framework for Bash.
```
Unfortunately, our clients don't support this rendering of links. Alternatively you can list it inside brackets.


```suggestion
> Bash Automated Testing System is a TAP (https://testanything.org/) compliant testing framework for Bash.
```

Reply:
> Why not in angled brackets "<>" as in the more information links?

Yeah, This should be enclosed inside an angle bracket. I forgot to add it while making the suggestion.